person.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'nameable'
+require_relative 'rental'
+
+class Person < Nameable
+  attr_reader :id, :parent_permission, :rentals
+  attr_accessor :name, :age
+
+  def initialize(age, parent_permission, name = 'Unknown')
+    super()
+    @id = generate_id
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly you have error on this part because never `generate` all kind of `person` type(student, teacher). Please fix them 🙏🏼 

![image](https://github.com/Armynerh/oop_library/assets/63915024/28868ee5-e6cf-4dfb-83ac-d6e6ee0d2c69)

![image](https://github.com/Armynerh/oop_library/assets/63915024/fe1c82c1-c400-4181-819f-bf86c2c1570f)

Reply:
@mukhammadrizooff Thank you for your feedback. The code works perfectly on my end
![Screenshot 2023-10-14 at 10 46 03](https://github.com/Armynerh/oop_library/assets/88312845/99042a01-4883-4b7e-8dd4-15dc60eefc5c)
![Screenshot 2023-10-14 at 10 44 58](https://github.com/Armynerh/oop_library/assets/88312845/8da4f61e-cb50-4941-9c1e-c4a903d7e700)